### DISCUSSION TITLE
can a win32 target release be added?

### DISCUSSION THREAD

**Discussion by Nos78** [KDE/ghostwriter#874] [Q&A]
I'll really like to run ghostwriter as the text editor of choice on my Windows (32bit) tablet - but there are only 64bit binaries available.

I did have a go at building a win32 from source - my PC isn't configured to build KDE apps from within VC, and my Raspberry PI4 isn't configured to build Windows executables. So both choices for me is requiring a bit of hard work/reasearch...

In hindsight I might have found the former easier, but I plumped for trying to build from Raspbian buster, since QT5 and KDE are already there, so it's just a case of getting mingw set up for building a windows i686 target, surely?

**Nooooo** it's never that easy! Buster repo is behind on QT5, behind on ECM, etc, etc... I gave up after the fourth or fifth attempt of having to grab library X, Y or Z, CMAKE it, make & install it,... and now I've killed my motivation to build a win32 target, I thought I'd ask the community (hoping someone already has a toolchain ready to go?)

**Comment by roke-julian-lockhart** [comment]
> Now I've killed my motivation to build a win32 target, I thought I'd ask the community.

@Nos78, as [`reddit.com/r/kde/comments/92aji5/comment/o5ctekt`](https://www.reddit.com/r/kde/comments/92aji5/comment/o5ctekt/?context=3&utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) demonstrates, the support shall likely decrease as time elapses. If you don't, I doubt that anyone else shall. However, [`sourceforge.net/projects/kde-windows/files/`](https://sourceforge.net/projects/kde-windows/files/) might be a vaguely useful starting point. [^1]

[^1]: [`reactos.org/forum/viewtopic.php?p=45204&sid=5b7200d26765eb50fc5a2fd8457d92e6#p45204`](https://reactos.org/forum/viewtopic.php?p=45204&sid=5b7200d26765eb50fc5a2fd8457d92e6#p45204)
